File: fizzBuzz.py
#!/usr/local/bin/python3

print('~Welcome to FizzBuzz~')
for x in range(100):
    if x % 3 != 0 and x % 5 != 0:
        print(str(x), end = '')
    if x % 3 == 0:
        print('Fizz', end = '')
    if x % 5 == 0:
        print('Buzz', end = '')
    print('')

File: fizzbuzz.c
#include<stdio.h>
int main(void){
  int cnt;
  printf("~Welcome FizzBuzz~\n");
  for(cnt = 0; cnt < 100; cnt++){

    if(cnt % 3 != 0 && cnt % 5!= 0){
      printf("%d", cnt);
    }
    if(cnt % 3 == 0) 
      printf("Fizz"); 

    if(cnt % 5 == 0) 
      printf("Buzz");
    printf("\n");
  }
  return 0;
}
